netket/nn/symmetric_linear.py

```diff
@@ -272,6 +292,8 @@ def __call__(self, x: Array) -> Array:
             *kernel.shape[:4], self.n_cells
         )
 
+        # TODO: the batch ordering should be revised: batch dimensions should
+        # be leading
```

Comment:
Wdym?

Reply:
for optimal performance batch dimensions should be the first on both arrays.
Jax does that automatically in many cases but maybe we can outsmart jax by changing the parameter shape.

I left it there just to remind us that this might be worth re-checking..

Reply:
After the previous thing with dimension permuting, it seems to me that XLA optimises this sort of thing better than we can